Shared/Reusable/ListButton.swift

```diff
@@ -12,18 +12,29 @@ import SwiftUI
 struct ListButton: View {
     var image: Image
     var text: String
-    var action: () -> Void
+    var action: () -> Void = {}
     
     var body: some View {
         Button(action: action) {
             HStack(spacing: 10) {
-                #if os(iOS)
-                HStack { image.font(.system(size: 20)) }
-                    .frame(width: 30)
+                image.font(.system(size: 20))
+                
                 Text(text)
-                #endif
```

Comment:
Here can we leave the HStack layout previously used as it is purposefully used to align the images on iOS?

Reply:
Okay 👍